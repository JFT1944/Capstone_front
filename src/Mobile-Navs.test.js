
import React from "react";
import { render, fireEvent, getByTestId } from "@testing-library/react";
import MobileNav from "./Mobile-Nav";
import { BrowserRouter, MemoryRouter } from "react-router-dom";


// write a smoke test for the home component

it('should appear in a smoketest', () => {
    <BrowserRouter>
    render(<MobileNav user={'user'} token={'token'}/>)
    </BrowserRouter>
})

// snapshot test
it('should match snapshot', () => {
    const {asFragment} = render(<MemoryRouter><MobileNav user={'user'} token={'token'}/></MemoryRouter>)
    expect(asFragment()).toMatchSnapshot()
})