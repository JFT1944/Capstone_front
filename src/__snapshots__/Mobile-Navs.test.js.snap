// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="logoM"
  >
    <span
      class="logo1"
    >
      Dispensa
      <span
        class="logo2"
      >
        .io
      </span>
    </span>
  </div>
  <div
    class="NavBar_holde-m"
  >
    <div
      style="height: 150px;"
    />
    <a
      class="navs"
      href="/part-form"
    >
      Add Single Ingredient
    </a>
    <a
      class="navs"
      href="/full-form"
    >
      Add Recipe Ingredients
    </a>
    <a
      class="navs"
      href="/recipes"
    >
      Recipes
    </a>
    <a
      class="navs"
      href="/pantry"
    >
      Pantry
    </a>
    <a
      class="navs"
      href="/shopping-list"
    >
      Shopping List
    </a>
    <a
      aria-current="page"
      class="navs active"
      href="/"
    >
      Log Out
    </a>
    <div
      style="height: 150px;"
    />
  </div>
</DocumentFragment>
`;
